notebooks/community/model_evaluation/automl_tabular_classification_model_evaluation.ipynb

```diff
@@ -0,0 +1,1375 @@
+{
```

Comment:
<p>to evaluate an AutoML classification* model. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>done.</p>